antarest/core/maintenance/repository.py

```diff
@@ -0,0 +1,42 @@
+from typing import Optional, List
+
+from antarest.core.configdata.model import ConfigData, ConfigDataAppKeys
+from antarest.core.configdata.repository import ConfigDataRepository
+
+
+class MaintenanceRepository(ConfigDataRepository):
+    def save_maintenance_mode(self, owner: int, mode: str) -> None:
```

Comment:
here you can remove the owner parameter and hardcode the owner to DEFAULT_ADMIN_USER.id since the maintenance mode is a special config data only set by the admin user.